db/schema.rb

```diff
@@ -425,6 +498,19 @@
     t.datetime "updated_at", null: false
   end
 
+  create_table "friendly_id_slugs", force: true do |t|
```

Comment:
I think that's because it's a generated table from the friendly_id module which provides slugs instead of IDs in paths. Good catch, I'll see if I can make it namespace properly.